File: Python/functions/no checking/neon.py
#Neon No.
def check(n):
    x=n**2
    s=0
    while x>0:
        s+=x%10
        x//=10        
    return s
n=int(input("Enter the no. = "))
if n==check(n):
    print("Neon No.")
else:
    print("Not Neon No.")

File: C Programms/No. Checking/neon.c
//WAP to check Neon no.
#include<stdio.h>
void main()
{
    int n,sq,s=0;
    printf("Enter the no. = ");
    scanf("%d",&n);
    for(sq=n*n;sq!=0;sq/=10)
        s+=sq%10;
    (s==n)?printf("%d is a Neon no.",n):printf("%d is not a Neon no.",n);
}